mk/PX4/ROMFS/defaults.parm

```diff
@@ -57,6 +57,7 @@ LOG_BITMASK, 131070.000000
 LOG_BACKEND_TYPE, 3.00000
 MNT_ANGMAX_TIL, 0.000000
 MNT_ANGMIN_TIL, -9000.000000
+#MNT_DEFLT_MODE, 1.0
```

Comment:
Yes. With the current shotmanager build it's irrelevant anyway (shotmanager sets RC_TARGETING at boot), so this would probably cause confusion at the moment if it were uncommented. It's there so we don't forget when everything comes together with the Qbert mode switching state logic.